mozilla/blurts-server
# MNTOR-2216: FxA webhook deletion should remove OneRep data

# References: 
Jira: MNTOR-2216




# Description
* Modify FxA webhook deletion to include OneRep removal
* Cascade delete for one rep tables etc.


# Screenshot (if applicable)

Not applicable.

# How to test
In heroku or stage, try deleting the user in fxa and see it the event logged and triggering a cascade delete. Verify that existing OneRep records don't exist anymore for that subscriber 


# Checklist (Definition of Done)
